### PR TITLE
Prevented user from following themselves

### PR DESCRIPTION
**Pull request by 0phelia36**

### CODE REVIEW

**Review by artbrock — APPROVED** [review]
It all looks pretty good to me. Thanks for contributing the PR!

I'm going to touch base with @jost-s before merging and make sure I'm not gonna mess up whatever he's working on next.

-art

File: tests/src/clutter/mews.ts [hunk 1]
```diff
@@ -122,6 +122,18 @@ export default (orchestrator: Orchestrator<any>) =>
     mews = await bob.call("mews", "mews_feed", { option: "" });
     t.equal(mews.length, 0);
 
+    try {
+      await bob.call("mews", "follow", bobPubKey);
+      await sleep(777);
```
Comment (jost-s) [inline tests/src/clutter/mews.ts:127]: This and the following line are never reached, because `bob.call(...)` throws and error and the error is caught.

**Review by jost-s — APPROVED** [review]
Thanks for bringing this back and adding the back end validation @0phelia36. It got lost during some refactoring, I guess 😲 

This doesn't interfere with me, @artbrock.

Please run `npm run lint` before merging.

File: tests/src/clutter/mews.ts (hunk 1 above)
Comment (0phelia36) [reply tests/src/clutter/mews.ts]: I removed the sleep line, but the t.fail() is still needed since if it doesn't throw an error the test should fail.

Comment (jost-s) [reply tests/src/clutter/mews.ts]: Ah, true 👍

**Pull request merged**